hugo/lib/createArticle.js

```diff
@@ -23,16 +15,19 @@ module.exports = (entry) => {
   let headerPhotoInfo = content.headerPhoto.fields;
 
   // Grab Author information
-  let {bio, name} = content.author.fields;
+  let { bio, name, twitter, facebook, instagram } = content.author.fields;
```

Comment:
I don't know off-hand if anyone's actually got one set, but I know `linkedin` is also an available field for authors to set. We should probably handle it just in case.